theano/tensor/blas.py

```diff
@@ -2134,6 +2134,10 @@ def c_code(self, node, name, inp, out, sub):
         _z, = out
         fail = sub["fail"]
 
+        if not config.blas.ldflags:
+            return super(BatchedDot, self).c_code(node, name,
+                                                  inp, out, sub)
```

Comment:
This don't raise an error. It just skip the c code for this op. It have a perform, so it won't cause crash later.